src/main/java/com/example/may/security/LocalSecurityConfiguration.java

```diff
@@ -0,0 +1,46 @@
+package com.example.may.security;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.core.userdetails.User;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.provisioning.InMemoryUserDetailsManager;
+import org.springframework.security.web.SecurityFilterChain;
+
+/**
+ * @author Evelina Tubalets
+ */
+@Profile({"default", "local"})
```

Comment:
**default** is not required here, defining only local profile means the same
`@Profile("local") == @Profile({"default", "local"})`